dizoo/dmc2gym/envs/dmc2gym_env.py

```diff
@@ -0,0 +1,169 @@
+from ding.envs import BaseEnv, BaseEnvTimestep
+from ding.torch_utils import to_ndarray
+from ding.utils import ENV_REGISTRY
+import dmc2gym
+
+
+def dmc2gym_observation_space(dim, minimum=-np.inf, maximum=np.inf, dytpe=np.float32) -> Box:
+    return Box(np.repeat(minimum, dim).astype(dytpe), np.repeat(maximum, dim).astype(dytpe), dtype=dytpe)
+
+
+def dmc2gym_state_space(dim, minimum=-np.inf, maximum=np.inf, dytpe=np.float32) -> Box:
+    return Box(np.repeat(minimum, dim).astype(dytpe), np.repeat(maximum, dim).astype(dytpe), dtype=dytpe)
+
+
+def dmc2gym_action_space(dim, minimum=-1, maximum=1, dytpe=np.float32) -> Box:
+    return Box(np.repeat(minimum, dim).astype(dytpe), np.repeat(maximum, dim).astype(dytpe), dtype=dytpe)
+
+
+def dmc2gym_reward_space(minimum=0, maximum=1) -> tuple:
+    return (
+        minimum,
+        maximum,
+    )
+
+
+dmc2gym_env_info = {
+    "ball_in_cup": {
+        "catch": {
+            "observation_space": dmc2gym_observation_space(8),
+            "state_space": dmc2gym_state_space(8),
+            "action_space": dmc2gym_action_space(2),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "cartpole": {
+        "balance": {
+            "observation_space": dmc2gym_observation_space(8),
+            "state_space": dmc2gym_state_space(8),
+            "action_space": dmc2gym_action_space(1),
+            "reward_space": dmc2gym_reward_space()
+        },
+        "swingup": {
+            "observation_space": dmc2gym_observation_space(8),
+            "state_space": dmc2gym_state_space(8),
+            "action_space": dmc2gym_action_space(1),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "cheetah": {
+        "run": {
+            "observation_space": dmc2gym_observation_space(17),
+            "state_space": dmc2gym_state_space(17),
+            "action_space": dmc2gym_action_space(6),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "finger": {
+        "spin": {
+            "observation_space": dmc2gym_observation_space(9),
+            "state_space": dmc2gym_state_space(9),
+            "action_space": dmc2gym_action_space(1),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "reacher": {
+        "easy": {
+            "observation_space": dmc2gym_observation_space(6),
+            "state_space": dmc2gym_state_space(6),
+            "action_space": dmc2gym_action_space(2),
+            "reward_space": dmc2gym_reward_space()
+        }
+    },
+    "walker": {
+        "walk": {
+            "observation_space": dmc2gym_observation_space(24),
+            "state_space": dmc2gym_state_space(24),
+            "action_space": dmc2gym_action_space(6),
+            "reward_space": dmc2gym_reward_space()
+        }
+    }
+}
+
+
+@ENV_REGISTRY.register('dmc2gym')
+class DMC2GymEnv(BaseEnv):
+
+    def __init__(self, cfg: dict = {}) -> None:
+        assert cfg.domain_name in dmc2gym_env_info
```

Comment:
`assert cfg.domain_name in dmc2gym_env_info, '{}/{}'.format(cfg.domain_name, dmc2gym_env_info.keys())`